### PR TITLE
Unify instructions in nami modules documentation

### PR DESCRIPTION
**Pull request by raquel-campuzano**
-Add components as persistent volumes
-In order to be consistent, some sentences are changed by taking
Wordpress as a model.

### CODE REVIEW

**Comment by prydonius** [comment]
woo thanks @raquel-campuzano! lgtm apart from the typo @jotadrilo spotted!

**Comment by raquel-campuzano** [comment]
My pleasure! Thank you both. Please let me know any issue, typo or lack of content you find. Feel free to create new tasks for keeping our repository updated!

File: README.md [hunk 1]
```diff
@@ -82,56 +82,69 @@ Then you can access your application at http://your-ip/
...
+
+To avoid inadvertent removal of these volumes you can [mount host directories as data volumes](https://docs.docker.com/engine/tutorials/dockervolumes/). Alternatively you can make use of volume plugins to host the volume data.
 
 > **Note!** If you have already started using your application, follow the steps on [backing](#backing-up-your-application) up to pull the data from your running container down to your host.
 
-### Mount persistent folders in the host using docker-compose
+### Mount host directories as data volumes with Docker Compose
 
-This requires a sightly modification from the template previously shown:
+TThis requires a minor change to the `docker-compose.yml` template previously shown:
```
Comment (sameersbn) [inline README.md]: typo!

File: README.md [hunk 2]
```diff
@@ -58,21 +58,21 @@ If you want to run the application manually instead of using docker-compose, the
 
 1. Create a new network for the application and the database:
 
-  ```
+  ```bash
   $ docker network create magento_network
```
Comment (sameersbn) [inline README.md]: please name this network `magento-tier` so that it matches the instructions from other sections.

Comment (raquel-campuzano) [inline README.md]: Done in commit fae21ae84ae44f290fe407e275c3f99237592ca2

**Pull request merged**